k9mail/src/main/java/com/fsck/k9/notification/NotificationIds.java

```diff
@@ -0,0 +1,41 @@
+package com.fsck.k9.notification;
+
+
+import com.fsck.k9.Account;
+
+
+class NotificationIds {
+    private static final int MAX_NUMBER_OF_ACCOUNTS = 1000;
+    private static final int DEFAULT_NOTIFICATION_OFFSET = 0;
+
+    private static final int BASE_ID_FETCHING_MAIL = -5000;
+    private static final int BASE_ID_SEND_FAILED_NOTIFICATION = -1500;
+    private static final int BASE_ID_CERTIFICATE_ERROR_INCOMING = -2000;
+    private static final int BASE_ID_CERTIFICATE_ERROR_OUTGOING = -2500;
```

Comment:
what if I have an account number of 501? shouldn't these have a distance of at least MAX_NUMBER_OF_ACCOUNTS?